src/SamplesApp/UITests.Shared/Windows_UI_Input/GestureRecognizerTests/PointersEvents.xaml.cs

```diff
@@ -191,6 +194,8 @@ private void SetupEvents(FrameworkElement target, bool handledToo, bool allEvent
 				target.AddHandler(TappedEvent, _logTapped, handledToo);
 			if (allEvents || _gestureDoubleTapped.IsOn)
 				target.AddHandler(DoubleTappedEvent, _logDoubleTapped, handledToo);
+			if (allEvents || _gestureRightTapped.IsOn)
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Add curly braces around the nested statement(s) in this 'if' block.](https://app.codacy.com/gh/unoplatform/uno/pullRequest?prid=4826514)